partner_invoicing_mode_at_shipping/__manifest__.py

```diff
@@ -12,6 +12,9 @@
         "data/queue_job_data.xml",
         "views/res_partner.xml",
     ],
-    "depends": ["account", "partner_invoicing_mode", "queue_job", "stock"],
+    "depends": ["account", "partner_invoicing_mode", "queue_job", "sale_stock"],
```

Comment:
No need for this change.
- depends: `sale_stock` is a module with `auto_install = True` and depends on the module `sale`, `stock`. And in `partner_invoicing_mode` it depends on `sale`. So `sale_stock` is already installed
```suggestion
    "depends": ["account", "partner_invoicing_mode", "queue_job", "stock"],
```